cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -196,7 +196,7 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         p("include::database/all-management-syntax.asciidoc[]")
 
         p(
-          """For example, granting the ability to create indexes, constraints, labels, relationship types and property names on the database `neo4j` to the role `regularUsers` is done using the following query.""".stripMargin)
+          """For example, granting the ability to access, start and stop all databases and create indexes, constraints, labels, relationship types and property names on the database `neo4j` to the role `regularUsers` is done using the following query.""".stripMargin)
```

Comment:
If you are listing all privileges here I do not think it should say `For example`?

Reply:
I think the `For example` is because after is an example query? (specifically giving it on `neo4j` to `regularUsers`)